tools/apprunner-gradle-plugin/src/main/java/org/projectnessie/quarkus/gradle/QuarkusAppExtension.java

```diff
@@ -15,16 +15,41 @@
  */
 package org.projectnessie.quarkus.gradle;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import org.gradle.api.Project;
 import org.gradle.api.provider.MapProperty;
+import org.gradle.api.provider.Property;
 
+@SuppressWarnings("UnstableApiUsage") // omit warning about `Property`+`MapProperty`
 public class QuarkusAppExtension {
   private final MapProperty<String, Object> props;
+  private final Property<String> nativeBuilderImage;
+
+  private static Map<String, Object> defaultProps() {
+    Map<String, Object> map = new HashMap<>();
+    map.put("quarkus.http.test-port", 0);
+    return map;
+  }
+
   public QuarkusAppExtension(Project project) {
-    props = project.getObjects().mapProperty(String.class, Object.class);
+    props = project.getObjects().mapProperty(String.class, Object.class).convention(defaultProps());
+
+    // This is not doing anything with Docker or building a native image, just a quirk of Quarkus since 1.10.
+    nativeBuilderImage = project.getObjects().property(String.class).convention("quay.io/quarkus/ubi-quarkus-native-image:21.0.0-java11");
```

Comment:
does this need to be updated on every quarkus release?

Reply:
Don't know - but it's configurable. It's the quirk from the last Quarkus version bump.

Reply:
Thinking of the deploy doc. I think there are a handful of versions to update by hand in the gradle plugin. Might as well add this one to the list.